src/attributes/codegen.md

```diff
@@ -155,12 +155,32 @@ Feature     | Implicitly Enables | Description
 `aes`       | `sse2`   | [AES] --- Advanced Encryption Standard
 `avx`       | `sse4.2` | [AVX] --- Advanced Vector Extensions
 `avx2`      | `avx`    | [AVX2] --- Advanced Vector Extensions 2
+`avx512bf16`        | `avx512bw`                | [AVX512-BF16] --- Advanced Vector Extensions 512-bit - Bfloat16 Extensions
+`avx512bitalg`      | `avx512bw`                | [AVX512-BITALG] --- Advanced Vector Extensions 512-bit - Bit Algorithms
+`avx512bw`          | `avx512f`                 | [AVX512-BW] --- Advanced Vector Extensions 512-bit - Byte and Word Instructions
+`avx512cd`          | `avx512f`                 | [AVX512-CD] --- Advanced Vector Extensions 512-bit - Conflict Detection Instructions
+`avx512dq`          | `avx512f`                 | [AVX512-DQ] --- Advanced Vector Extensions 512-bit - Doubleword and Quadword Instructions
+`avx512f`           | `avx2`                    | [AVX512-F] --- Advanced Vector Extensions 512-bit - Foundation
```

Comment:
Should this include `fma` and `f16c`? 
https://github.com/rust-lang/rust/blob/a15cce2690e8fab72422515c9dc02c6fbc506733/compiler/rustc_target/src/target_features.rs#L400